### PR TITLE
remove the score from the default google result page

### PR DESCRIPTION
**Pull request by eishay**

### CODE REVIEW

File: chrome/google_inject.html [hunk 1]
```diff
@@ -11,7 +11,7 @@ <h3 class="r">Kifi Trusted Results</h3>
       <h3 class="r"><a href="{{bookmark.url}}">{{bookmark.title}}</a></h3>
       <div class="vspib" aria-label="Result details" role="button" tabindex="0"></div>
       <div class="s">
-        <div class="f kv"><cite>{{{score}}}{{displayUrl}}</cite>
```
Comment (andrewconner) [inline chrome/google_inject.html:14]: Was this meant to be removed, with the bugfix in google_inject.js?

Comment (eishay) [inline chrome/google_inject.html:14]: Probably a mistake, good catch.

Comment (eishay) [inline chrome/google_inject.html:14]: when i re-add it the score keeps on showing, wonder why

Comment (andrewconner) [inline chrome/google_inject.html:14]: Is the config not being reset / saved? I'll trace through it Monday. Keep it out if you prefer it being clean for now.

By the way, I've added a lot of new CSS rules that help on the other commit. They fix most of the issues I've found so far.

Comment (eishay) [inline chrome/google_inject.html:14]: re-added the score

**Comment by eishay** [comment]
updated pull request:
1. use a consistent config and local storage keys.
2. fix score display bug

**Pull request merged**